crates/l2/sdk/src/sdk.rs

```diff
@@ -161,44 +158,26 @@ pub async fn withdraw(
 }
 
 pub async fn claim_withdraw(
-    l2_withdrawal_tx_hash: H256,
     amount: U256,
+    tx_hash: H256,
```

Comment:
Let's keep this name.
```suggestion
    l2_withdrawal_tx_hash: H256,
```

Reply:
Addressed in [98a4169](https://github.com/lambdaclass/ethrex/pull/2397/commits/98a4169849a845200e06cd91cbdb350cfb0655c5)!